pkg/features/kube_features.go

```diff
@@ -180,6 +186,7 @@ var defaultFeatureGates = map[featuregate.Feature]featuregate.FeatureSpec{
 	ExposeFlavorsInLocalQueue:           {Default: true, PreRelease: featuregate.Beta},
 	AdmissionCheckValidationRules:       {Default: false, PreRelease: featuregate.Deprecated},
 	KeepQuotaForProvReqRetry:            {Default: false, PreRelease: featuregate.Deprecated},
+	LocalQueueDefaulting:                {Default: false, PreRelease: featuregate.Alpha},
```

Comment:
Please also update [The currently supported features table](https://github.com/kubernetes-sigs/kueue/blob/a2e7c62e846791759a5a7914e0042221a682aa57/site/content/en/docs/installation/_index.md?plain=1#L234)